### PR TITLE
Add styles for links

### PR DESCRIPTION
**Pull request by Ladsgroup**
We added links to descriptions and now we need to add styles for that.

Based on https://www.figma.com/file/gi3JYwSUQYhKS0mHwK25cQwi/WiKit?node-id=2510%3A3550
and tokens

### CODE REVIEW

**Comment by sai-san** [comment]
Just checked locally and it looks good. Thanks, Amir! 🙏  I left a tiny reminder inline. Let me know what you think

File: src/components/QueryBuilder.vue [hunk 1]
```diff
@@ -75,6 +75,19 @@ export default Vue.extend( {
 </script>
 
 <style scoped lang="scss">
+a {
```
Comment (sai-san) [inline src/components/QueryBuilder.vue:79]:
```suggestion
<!-- TODO replace with link component once available-->
a {
```

**Comment by Ladsgroup** [comment]
> Just checked locally and it looks good. Thanks, Amir! pray I left a tiny reminder inline. Let me know what you think

Done. Thanks!

File: src/components/QueryBuilder.vue (hunk 1 above)
Comment (Ladsgroup) [reply src/components/QueryBuilder.vue:79]: This actually fails because this is not valid css :(

File: src/components/QueryBuilder.vue [hunk 2]
```diff
@@ -75,6 +75,20 @@ export default Vue.extend( {
 </script>
 
 <style scoped lang="scss">
+<!-- TODO replace with link component once available-->
```
Comment (micgro42) [inline src/components/QueryBuilder.vue]: Needs to be an CSS comment, not an HTML comment. I failed to spot that earlier

```suggestion
// TODO replace with link component once available
```

**Review by micgro42 — CHANGES_REQUESTED** [review]

File: src/components/QueryBuilder.vue (hunk 2 above)
Comment (sai-san) [reply src/components/QueryBuilder.vue]: agh! of course. sorry about that!

**Review by micgro42 — APPROVED** [review]

**Pull request merged**